Mozilla/Firefox.download.recipe

```diff
@@ -20,9 +20,11 @@ combination is offered.</string>
         <string>en_US</string>
         <key>NAME</key>
         <string>Firefox</string>
+        <key>DISABLE_CODE_SIGNATURE_VERIFICATION</key>
+        <string>False</string>
```

Comment:
Shouldn't this be ```<false />``` instead of ```<string>False</string>```? I'm not getting any verification done with this...